tests/test_floors.py

```diff
@@ -1,12 +1,34 @@
+import pytest
+
+
 def test_list(client):
     assert client.get("/floors/").status_code == 200
 
-def test_create(client):
-    assert client.get("/floors/create").status_code == 200    
 
-def test_edit(client):
-    assert client.get("/floors/edit/1").status_code == 200     
+@pytest.mark.parametrize("path", (
+    "floors/create",
+    "/floors/edit/1",
+    "/floors/delete",
+))
+def test_login_required(client, path):
+    response = client.post(path)
+    assert response.headers["Location"] == "http://localhost/auth/login"
```

Comment:
@bkuzmic I think the best way is to use `url_for` here
http://flask.pocoo.org/docs/1.0/api/#flask.url_for

could we use it here?

Reply:
@vladarefiev Yes, we can. I'll do that.